### ISSUE TITLE
Signal notifications getting ignored

### ISSUE THREAD

**Issue by eddic00l** [Freeyourgadget/Gadgetbridge#227]
Hello,

push notifications of Signal messages only work for normal SMS for me, encrypted Signal messages aren't recognized by Gadgetbridge it seems.

I'm using the latest 0.7.4 version with a Pebble Steel on Firmware 3.9.2.

Every other message is arriving.

**Comment by danielegobbetti** [comment]
Hi, I noticed signal does not send notifications if it is left open (ie you lock the phone with signal app in foreground). Could this be the case?

**Comment by eddic00l** [comment]
I didn't enable the lockscreen for Signal, notifications are normally shown at my phone.

**Comment by ashimokawa** [comment]
@eddic00l 
Which android version do you use? At least with CM13 I can prevent Signal notifications to be shown in the lock screen which might cause the problem.

Do further debug, you could enable generic notifications and check the "also when screen is on" checkmark. Then in the debug activity create a test notification. If that works leave the screen on and tell someone to send you a Signal message. If that also works it might be some lockscreen issue.

**Comment by danielegobbetti** [comment]
@eddic00l can you please check again if gadgetbridge's access to generic notifications is enabled (GB-> settings -> generic notifications)? I can only reproduce this issue by turning this off.

**Comment by Samarium151** [comment]
I had the same issue with Miband 1s  FW: 4.15.11.19, running android 5.1 (xprivacy is installed as well) Gadget bridge 0.8.0
signal messages no notification,  SMS yes no problem.  I use signal as my messaging client.  
k-9 email  notifications work fine

I tried enabling generic notifications. Turned on also when screen is on turned sms notifications to always.  Notification access is enabled .

I had to quit and restart gadgetbridge after changing settings , after that works correctly.
